.github/workflows/pre-release-check.yml

```diff
@@ -24,4 +24,4 @@ jobs:
       - name: set JVM opts
         run: scripts/gha_setup.sh
       - name: Run Dataflow jobs
-        run: sbt ${{ env.SBT_OPTS }} "scio-examples/IntegrationTest/runMain com.spotify.scio.examples.RunPreReleaseIT --runId=${{ github.run_id }}"
+        run: sbt ${{ env.SBT_OPTS }} "scio-examples/Test/runMain com.spotify.scio.examples.RunPreReleaseIT --runId=${{ github.run_id }}"
```

Comment:
left the `RunPreReleaseIT` in `scio-exaples` to not add this non scio module dependency to `integration`